src/BaGet.Core.Server/Configuration/ConfigureForwardedHeadersOptions.cs

```diff
@@ -1,15 +1,36 @@
+using System;
+using System.Collections.Generic;
+using BaGet.Core;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.HttpOverrides;
+using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 
+
 namespace BaGet.Configuration
 {
     public class ConfigureForwardedHeadersOptions : IConfigureOptions<ForwardedHeadersOptions>
     {
-        public void Configure(ForwardedHeadersOptions options)
+        private IOptions<BaGetOptions> _options;
+        private ILogger<ConfigureForwardedHeadersOptions> _logger;
+        public ConfigureForwardedHeadersOptions(IOptions<BaGetOptions> bagetOptions, ILogger<ConfigureForwardedHeadersOptions> logger)
         {
-            options.ForwardedHeaders = ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto;
+            _options = bagetOptions;
+            _logger = logger;
+        }
 
+        public void Configure(ForwardedHeadersOptions options)
+        {
+            if (!string.IsNullOrEmpty(_options.Value.HostUrl))
+            {
+                _logger.LogDebug("Host Url: " + _options.Value.HostUrl);
```

Comment:
It's better to let the logging framework format the log message. This improves performance if the log level is set higher than debug (as the log will just be dropped). Example:

```suggestion
                _logger.LogDebug("Allowing host URL {HostUrl}", _options.Value.HostUrl);
```